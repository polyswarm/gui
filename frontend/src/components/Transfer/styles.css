.Transfer {
    background-color: #f5f5f5;
    background-image: url(gray_hexgrid_2x.png);
    background-size: 129px 149px;
    background-repeat: repeat;
    height: 100vh;
    left: 0;
    position: fixed;
    top: 0;
    width: 100vw;
}