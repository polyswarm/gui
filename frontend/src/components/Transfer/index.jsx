// Vendor imports
import React, { Component } from 'react';
import PropTypes from 'prop-types';
// Project imports
import Header from '../Header';
import NectarField from '../NectarField';
import ChainInfo from '../ChainInfo';
import TransferForm from '../TransferForm';
// Component imports
import './styles.css';

class Transfer extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    const {props: {nectar}} = this;
    return(
        <div className='Transfer'>
          <Header />
          <div className='Transfer-Content'>
            <NectarField
              onChange={this.onNectarChange}
              nectar={nectar}/>
            <div >
              <ChainInfo />
              <ChainInfo />
            </div>
            <TransferForm />
          </div>
        </div>
    );
  }

}
Transfer.proptypes = {
  nectar: PropTypes.string
}
export default Transfer;