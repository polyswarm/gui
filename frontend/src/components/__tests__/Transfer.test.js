import React from 'react';
import {render} from 'enzyme';
import {renderToJson} from 'enzyme-to-json';
import Transfer from '../Transfer';

it('renders without crashing', () => {
    const wrapper = render(<Transfer />);
  expect(renderToJson(wrapper)).toMatchSnapshot();
});