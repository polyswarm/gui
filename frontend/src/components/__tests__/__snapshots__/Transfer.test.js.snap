// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  class="Transfer"
>
  <header
    class="Header"
  >
    PolySwarm Relay
  </header>
  <div
    class="Transfer-Content"
  >
    <div
      class="NectarField"
    >
      <label
        class="NectarField-Label"
        for="nectar"
      >
        Enter nectar
      </label>
      <input
        class="NectarField-Input"
        id="nectar"
        type="text"
        value=""
      >
        
      </input>
    </div>
    <div>
      <div>
        
      </div>
      <div>
        
      </div>
    </div>
    <div>
      
    </div>
  </div>
</div>
`;
